host/3.0/publish-appservice-stage.yml

```diff
@@ -49,12 +49,15 @@ steps:
 
   - bash: |
       docker pull $SOURCE_REGISTRY/java:$(PrivateVersion)-java8-appservice
+      docker pull $SOURCE_REGISTRY/java:$(PrivateVersion)-java8-coretools
 
       docker tag $SOURCE_REGISTRY/java:$(PrivateVersion)-java8-appservice $TARGET_REGISTRY/java:$(TargetVersion)-appservice-stage$(StageNumber)
       docker tag $SOURCE_REGISTRY/java:$(PrivateVersion)-java8-appservice $TARGET_REGISTRY/java:$(TargetVersion)-java8-appservice-stage$(StageNumber)
+      docker tag $SOURCE_REGISTRY/java:$(PrivateVersion)-java8-coretools $TARGET_REGISTRY/java:$(TargetVersion)-java8-coretools-stage$(StageNumber)
 
       docker push $TARGET_REGISTRY/java:$(TargetVersion)-appservice-stage$(StageNumber)
       docker push $TARGET_REGISTRY/java:$(TargetVersion)-java8-appservice-stage$(StageNumber)
+      docker push $TARGET_REGISTRY/java:$(TargetVersion)-java8-coretools-stage$(StageNumber)
```

Comment:
I don't think we need to stage updates to this image, right? might be enough to just have it in the publish.yml file.